config/networking-networkpolicy.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: knative-serving-user-container-policy
+  namespace: default
```

Comment:
Is this applied to all namespaces automatically? Don't we need this in every namespace that hosts Knative Revisions ultimately?

Reply:
It doesn't cover all namespaces so I'll have to make updates to cover all. I think we wanted to see if the policy works with the tests first, or if it breaks anything. Since it only covers default we can either think about recommending that users apply the networkpolicy for new namespaces or back to the per revision networkpolicy.

Reply:
I'd perhaps move this to `serving-tests` to get a real test run with this applying to revisions.